.github/CODEOWNERS

```diff
@@ -1,3 +1,3 @@
 # Every request must be reviewed and accepted by:
 
-* @gdcorp-action-public-forks/cloud-foundations 
+*               @gdcorp-engineering/cloud-foundations
```

Comment:
You can't reference GitHub teams across organizations. It has to be the team in `gdcorp-action-public-forks`